samples/Move/Main.move

```diff
@@ -0,0 +1,110 @@
+address admin {
+module Main {
+    use StarcoinFramework::Account;
+    use StarcoinFramework::Signer;
+    use StarcoinFramework::Token;
+    use StarcoinFramework::Event;
+    use SFC::PseudoRandom;
+
+    struct Bank<phantom T: store> has store, key {
+        bank: Token::Token<T>
+    }
+
```

Comment:
I think we should use a more simple example, mainly used to show the syntax characteristics of Move.

Reply:
👌🏻